tests/types/dateTime.spec.ts

```diff
@@ -0,0 +1,74 @@
+import {StrongDateTime, makeDateTime} from '../../src/types/dateTime';
```

Comment:
Similar to the test filename change below to all lowercase separated by hyphens, this import will need to be updated when the `dateTime` filename is changed to `date-time`.

- [ ] Change import from `'../../src/types/dateTime` to `../../src/types/date-time'`